colossalai/inference/README.md

```diff
@@ -90,25 +111,85 @@ cd /path/to/CollossalAI
 pip install -e .
 
 # install lightllm
-git clone https://github.com/ModelTC/lightllm 
+git clone https://github.com/ModelTC/lightllm
```

Comment:
These should be moved to dockerfile

Reply:
Rmove all dependencies. We need update the dockerfile later.